odl/contrib/datasets/README.md

```diff
@@ -21,6 +21,11 @@ Reference datasets with accompanying ODL geometries etc.
     CT data as provided by FIPS. The data is non-human and high resolution.
     * `walnut_data`
     * `lotus_root_data`
+
+
+    CT data as provided by Mayo Clinic. The data is human and high resolution. To access the data, see https://www.aapm.org/GrandChallenge/LowDoseCT/#registration
```

Comment:
Maybe you can also write a word or two on what you have to do to get the data (no details, just that it requires registration because it's potentially sensitive). I often find it frustrating to just be pointed to a website, only to find out 10 minutes later that I can't get it easily.

Reply:
I agree we should be explicit about that stuff